web/gds-user-ui/src/locales/en/messages.po

```diff
@@ -13,6 +13,25 @@ msgstr ""
 "Language-Team: \n"
 "Plural-Forms: \n"
 
+#: src/components/ReviewSubmit/index.tsx:192
+#~ msgid ""
+#~ " for MainNet registration. Your MainNet certificate will be issued upon review by\n"
+#~ "                  the validation team"
+#~ msgstr ""
+#~ " for MainNet registration. Your MainNet certificate will be issued upon review by\n"
+#~ "                  the validation team"
```

Comment:
@nwhite222 would you mind removing all the lines that start with `#~`? These have been removed from the web content and should no longer be in the .po files

Reply:
I've changed that chunk in the .po files - did you also want me to remove _all_ of the lines that start with "#: src", or just this chunk?